core/web/resolver/csa_keys_test.go

```diff
@@ -0,0 +1,131 @@
+package resolver
+
+import (
+	"encoding/json"
```

Comment:
Organise imports in the order of stdlib/external/internal packages separated by a new line

Reply:
Noted, I've modified Goland to group all `github.com/smartcontractkit/` imports together, same as using `goimports -local github.com/smartcontractkit/`.